workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1,58 +1,57 @@
-rule download:
-    message: "Downloading metadata and fasta files from S3"
+rule download_sequences:
+    message: "Downloading sequences from S3 bucket {params.s3_bucket}"
     output:
-        sequences = config["sequences"],
-        metadata = config["metadata"]
+        sequences = config["sequences"]
     conda: config["conda_environment"]
+    params:
+        s3_bucket = config["S3_BUCKET"]
     shell:
         """
-        aws s3 cp s3://nextstrain-ncov-private/metadata.tsv.gz - | gunzip -cq >{output.metadata:q}
-        aws s3 cp s3://nextstrain-ncov-private/sequences.fasta.gz - | gunzip -cq > {output.sequences:q}
+        aws s3 cp s3://{params.s3_bucket}/sequences.fasta.gz - | gunzip -cq > {output.sequences:q}
```

Comment:
There's a bit of inconsistency here where the other rules use `{params.compression}` but this uses gzip. Switching would require us modifying the ingest scripts which is for another day.

Reply:
I kept these files as gzip since they are shared via GISAID while the remainder is just for us...

Reply:
Ahh. good point.